src/Oscoin/Crypto/Hash/RealWorld.hs

```diff
@@ -150,6 +170,9 @@ instance Show.Show (Hash Crypto) where
                   . convert
                   $ d
 
+instance Show.Show (ShortHash Crypto) where
+    show = T.unpack . F.sformat ("0x" F.% formatShortHash)
```

Comment:
Apparently there is a `hex` formatter inside `Formatting.Formatters` that should allegedly do the same thing.

Reply:
Looks like it's not what we want (expects an integral) :( - I moved the prefix to `formatShortHash` to clean things up instead.